app/routes/goal_routes.py

```diff
@@ -1 +1,108 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from ..models.goal import Goal
+from ..models.task import Task
+from ..db import db
+
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+def validate_goal(goal_id):
+    try:
+        gid = int(goal_id)
```

Comment:
We can reassign the value of `goal_id` here instead of creating a new variable.

I'd prefer reassignment because `gid` is concise but not as descriptive as `goal_id`. We also do not need `goal_id` to continue referencing the string version of this number so it's safe to reassign the variable to be the int value. 

```suggestion
        goal_id = int(goal_id)
```

We do the same thing [here in Flasky](https://github.com/Ada-C23/flasky/blob/c23-live-code/app/routes/route_utilities.py#L6).
```py
model_id = int(model_id)